frontend/packages/core/src/Table/table.tsx

```diff
@@ -79,13 +79,14 @@ export const Table: React.FC<TableProps> = ({ headings, children, ...props }) =>
 
 export interface TableRowProps extends Pick<MuiTableRowProps, "onClick"> {
   children?: React.ReactNode;
+  defaultValue?: any;
 }
 
-export const TableRow = ({ children = [], onClick }: TableRowProps) => (
+export const TableRow = ({ children = [], onClick, defaultValue }: TableRowProps) => (
```

Comment:
What's the use case for specified a default value to the table row directly instead of handling it in the child?

Reply:
The use case is to prevent having to write out the default value for each child, which results in lots of copying and pasting.  But maybe I'm going about this in the wrong way.